README.md

```diff
@@ -106,6 +117,25 @@ mysql.query(SQL`SELECT * FROM `.append(mysql.escapeId(someUserInput)).append(SQL
 pg.query(SQL`SELECT * FROM `.append(pg.escapeIdentifier(someUserInput)).append(SQL` WHERE name = ${book} ORDER BY ${column} `).append(order))
 ```
 
+## Raw values with the RAW function / template tag
+Values returned from the RAW function (either by invocation on a string parameter or by use as a template tag) will be
+inserted raw when passed to the template literal or the `append()` function.
+
+```js
+const RAW = SQL.RAW
+
+const table = 'books'
+const column1 = 'name'
+const column2 = 'author'
+
+const columns = RAW`${column1}, ${column2}`
+const tableName = RAW(table)
```

Comment:
calling a function with uppercase letters seems very unconventional to me. Template string tags seem to also move into the direction of using lowercase (there wasn't a clear convention when I started this module). I'd prefer if it was also exported as a lowercase name and we use that in the readme. We should probably add a lowercase `sql` tag too.

Reply:
@felixfbecker I agree, I was aliasing it to lowercase when using it.